directjngine/src/test/java/com/softwarementors/extjs/djn/jscodegen/CodeFileGeneratorTest.java

```diff
@@ -0,0 +1,34 @@
+package com.softwarementors.extjs.djn.jscodegen;
```

Comment:
I believe we need to retain the same license header as used for other source files, i.e. https://github.com/sonatype/directjngine/blob/master/directjngine/src/test/java/com/softwarementors/extjs/djn/gson/DefaultGsonBuilderConfiguratorTest.java

Reply:
Oops I missed it. 😅 
Applied in d7a1d5c